src/gui/presets.cpp

```diff
@@ -931,6 +986,95 @@ void FurnaceGUI::initSystemPresets() {
       0
     }
   ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Capcom CPS-1", { 
+      DIV_SYSTEM_YM2151, 64, 0, 2,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Jaleco Mega System 1", {
+      DIV_SYSTEM_YM2151, 64, 0, 2,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "NMK Arcade", {
+      DIV_SYSTEM_OPN, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Data East Arcade", {
+      DIV_SYSTEM_OPN, 64, 0, 0,
+      DIV_SYSTEM_OPL2, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Kaneko Arcade", {
+      DIV_SYSTEM_AY8910, 64, 0, 1,
+      DIV_SYSTEM_AY8910, 64, 0, 1,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Tecmo Arcade", {
```

Comment:
Ninja Ryukenden(a.k.a Shadow Warriors, Ninja Gaiden), Wild Fang(a.k.a Tecmo Knights), Raiga - Strato Fighter uses this configuration. 4MHz for YM2203s, 1MHz for MSM6295.

Reply:
Problem is, naming it "Tecmo Arcade (Ninja Ryukenden et al.)" doesn't seem OK for me...